src/TinkerServiceProvider.php

```diff
@@ -27,9 +27,11 @@ public function register()
 
         $this->commands(['command.tinker']);
 
-        $this->publishes([
-            __DIR__.'/../config/tinker.php' => $this->app['path.config'].DIRECTORY_SEPARATOR.'tinker.php',
-        ]);
+        if ($this->app->runningInConsole()) {
```

Comment:
Still doesn't fix lumen support. Please see my blog post for what I mean. https://blog.bugsnag.com/designing-laravel-packages/. Feel free to lift the code from https://github.com/GrahamCampbell/Laravel-Example/blob/master/src/CatServiceProvider.php#L12-L23.